airflow/contrib/hooks/qubole_hook.py

```diff
@@ -175,6 +178,25 @@ def get_jobs_id(self, ti):
             cmd_id = ti.xcom_pull(key="qbol_cmd_id", task_ids=self.task_id)
         Command.get_jobs_id(self.cls, cmd_id)
 
+    def get_redirect_url(self, task, dttm):
```

Comment:
Can we wrap a try/catch/final around this, so we are sure that the connection gets closed in case something goes wrong with the xcom operation?

Reply:
sure, will do

Reply:
Please call this something like get_deeplink